src/appComponents/twitterwall/singletweet.js

```diff
@@ -44,20 +44,12 @@ function tweet($timeout,$rootScope) {
         scope.changeTweet=changeTweet;
         scope.callchangetweet=callchangetweet;
         scope.even=false;
-        getObject();
-        scope.$on('array',function(args,message){
-            scope.array=message;
-        });
-        function getObject(){
-            scope.$watch('array',function(oldvalue,newvalue){
-            if(scope.array){
-            scope.selectedid=scope.array[1].id_str;
-            changeTweet();
-            }
-            })
+         
+        function callchangetweet(){
```

Comment:
@juslee ,@championswimmer , this is against the code structure i follow , "[sir john papa style](https://github.com/johnpapa/angular-styleguide) " , Should I ignore it.